### ISSUE TITLE
CODEFORCES: 277A - Learning Languages

### ISSUE THREAD

**Issue by dev-onejun** [dev-onejun/Algorithm-Study#10]
https://codeforces.com/problemset/problem/277/A

**Comment by dev-onejun** [comment]
## Idea

- bfs (node: employee, tree: connected if the employees have the same language)
- The number of the constructed tree is the answer

**Comment by dev-onejun** [comment]
Wrong answer on test 4

**Comment by dev-onejun** [comment]
## Idea 2

you should add the language what the employee learned so that the language learned employee can append at some tree.

**Comment by dev-onejun** [comment]
According to [the contest material](https://codeforces.com/blog/entry/6815),
The test 4 is a corner case which requires to think all employees do not know any kinds of languages.

Before that, the number of edges which make the constructed graph connected was the only answer of the problem.
However, the corner case should be handled separately.
If no one had the languages to do, all employees(node) should learn.
In other words, the number of nodes becomes the answer.

**Comment by dev-onejun** [comment]
## Submission

https://codeforces.com/contest/277/my
